extensions/terminal-suggest/src/completions/code-tunnel.ts

```diff
@@ -0,0 +1,216 @@
+	},
+	{
+		name: 'rename',
+		description: 'Rename the name of this machine associated with port forwarding service',
+		options: commonCLIOptions,
+		args: {
+			name: 'name',
+		},
+	},
+	{
+		name: 'status',
+		description: 'Print process usage and diagnostics information',
+		options: commonCLIOptions,
+	},
+	{
+		name: 'unregister',
+		description: 'Remove this machine\'s association with the port forwarding service',
+		options: commonCLIOptions,
+	},
+	{
+		name: 'user',
+		subcommands: [
+			{
+				name: 'login',
+				description: 'Log in to port forwarding service',
+				options: [...commonAuthOptions, ...commonCLIOptions],
+			},
+			{
+				name: 'logout',
+				description: 'Log out of port forwarding service',
+				options: commonCLIOptions,
+			},
+			{
+				name: 'show',
+				description: 'Show the account that\'s logged into port forwarding service',
+				options: commonCLIOptions,
+			},
+			{
+				name: 'help',
+				description: 'Print this message or the help of the given subcommand(s)',
+				subcommands: [
+					{ name: 'login', description: 'Log in to port forwarding service' },
+					{ name: 'logout', description: 'Log out of port forwarding service' },
+					{ name: 'show', description: 'Show the account that\'s logged into port forwarding service' },
+					{ name: 'help', description: 'Print this message or the help of the given subcommand(s)' },
+				],
+				options: commonCLIOptions,
+			},
+		],
+		options: commonCLIOptions,
+	},
+	{
+		name: 'service',
+		description: '(Preview) Manages the tunnel when installed as a system service,',
+		subcommands: [
+			{
+				name: 'install',
+				description: 'Installs or re-installs the tunnel service on the machine',
+				options: [
+					{
+						name: '--name',
+						description: 'Sets the machine name for port forwarding service',
+
+						args: {
+							name: 'name',
+
+						},
+					},
+					{
+						name: '--accept-server-license-terms',
+						description: 'If set, the user accepts the server license terms and the server will be started without a user prompt',
+					},
+					...commonCLIOptions,
+				],
+			},
+			{
+				name: 'uninstall',
+				description: 'Uninstalls and stops the tunnel service',
+				options: commonCLIOptions,
+			},
+			{
+				name: 'log',
+				description: 'Shows logs for the running service',
+				options: commonCLIOptions,
+			},
+			{
+				name: 'help',
+				description: 'Print this message or the help of the given subcommand(s)',
+				subcommands: [
+					{ name: 'install', description: 'Installs or re-installs the tunnel service on the machine' },
+					{ name: 'uninstall', description: 'Uninstalls and stops the tunnel service' },
+					{ name: 'log', description: 'Shows logs for the running service' },
+					{ name: 'help', description: 'Print this message or the help of the given subcommand(s)' },
+				],
+				options: commonCLIOptions
+			},
+		],
+		options: commonCLIOptions,
+	},
+	{
+		name: 'forward-internal',
```

Comment:
This shouldn't be publicly documented, it's just internally by the port forwarding service and is not for users to interact with directly